src/main/kotlin/com/terraformation/backend/funder/db/FundingEntityStore.kt

```diff
@@ -39,8 +41,7 @@ class FundingEntityStore(
                 FUNDING_ENTITIES.CREATED_TIME,
                 FUNDING_ENTITIES.MODIFIED_TIME,
                 PROJECTS.ID,
-                PROJECTS.NAME,
-                PROJECTS.ORGANIZATION_ID)
+            )
```

Comment:
Do we still need the join with `projects` here? The ID can come from `funding_entity_projects`.

This query is using the project name in its `ORDER BY` but that seems wrong since the project name could sort differently than the deal name. If we want to sort by the user-facing name, then we'll either need to have a subquery or a CTE to fetch the deal name as part of the query so we can sort on it, or we'll need to sort in the application code instead of in SQL. Either way, the join with `projects` isn't needed.

Reply:
Ah thanks for catching, I had meant to remove the PROJECTS join, and had forgotten that I was ordering by deal name. I might have to update the FE to order by deal name, though if what Tommy is talking about will actually happen then this PR might be deleted anyway